platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -35,6 +35,20 @@ The previous step will help you identify potential problems by issuing deprecati
 . Run `gradle wrapper --gradle-version {gradleVersion}` to update the project to {gradleVersion}.
 . Try to run the project and debug any errors using the <<troubleshooting.adoc#troubleshooting, Troubleshooting Guide>>.
 
+[[changes_8.8]]
+== Upgrading from 8.7 and earlier
+
+=== Deprecations
+
+[[deprecated_namers]]
```

Comment:
`deprecated_namers` looks inconsistent with the `DeprecationLogger.withUpgradeGuideSection(8, "deprecated_namer")` calls (note the extra `s` at the end)